azure-pipelines.yml

```diff
@@ -61,10 +61,14 @@ stages:
               imageName: 'ubuntu-16.04'
               ORG_GRADLE_PROJECT_TILEDB_S3: "ON"
               ORG_GRADLE_PROJECT_TILEDB_SERIALIZATION: "ON"
+              CFLAGS: "-march=core2 -mtune=haswell -mssse3"
```

Comment:
Let's do `-march=haswell`, even though we used `-march=core2` previously, setting the minimum to AVX2 will provide a good level of performance. If we have a user who needs `-march=core2` we can adjust, but lets aim for the performance benefits of AVX/AVX2.

Reply:
done!